### PR TITLE
Marshal and Guild buff

### PR DESCRIPTION
**Pull request by cdb-is-not-good**
New guildcraft gun! Abdication, a VERY nasty energy scattershot that requires a large investiture of time and materials to produce. Slightly lower damage than buckshot but with the ability to take all variety of mods.

Ironhammer:tm: breaching hammer now fits on armor,  does SIGNIFICANT damage now(can potentially take down r-walls with patience.) but has a significant click delay, lessening its effectiveness as a weapon(Sorry Mojave).

Dark Gorkas/gorka pants in the loadout.

## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/tools/hammer.dm [hunk 1]
```diff
@@ -112,13 +112,14 @@
 
 /obj/item/tool/hammer/ironhammer
 	name = "GmbH \"Ironhammer\" Breaching Hammer"
-	desc = "A modified sledgehammer produced by Seinmetall for police forces across Sol space. This tool can take down standard walls and if the user is strong enough, reinforced walls."
+	desc = "A modified sledgehammer produced by Seinmetall for police forces across Sol space.  "
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/tools/hammer.dm:115]:
```suggestion
	desc = "A modified sledgehammer produced by Seinmetall for police forces across Sol space."
```

Comment (cdb-is-not-good) [reply code/game/objects/items/weapons/tools/hammer.dm:115]: why? It was buffed explicitly to be able to take down r-walls, hence the addition of delay.

File: code/modules/projectiles/guns/energy/cog.dm [hunk 2]
```diff
@@ -25,3 +25,26 @@
...
 	return
+
+/obj/item/gun/energy/cog/shotgun
+	name = "\"Abdicator\" energy shotgun"
+	icon = 'icons/obj/guns/energy/abdication.dmi'
+	icon_state = "abdication"
+	item_state = null
+	item_charge_meter = TRUE
+	desc = "A shotgun that epitomizes the truth that the tallest of buildings and grandest achievments are built by digging the deepest into the earth, \
+	fires superheated-shot in a crimson splutter, as the Reductor rail rifle. , \
```
Comment (r4d6) [inline code/modules/projectiles/guns/energy/cog.dm:36]:
```suggestion
	fires superheated-shot in a crimson splutter, as the Reductor rail rifle. \
```

File: code/game/objects/items/weapons/tools/hammer.dm (hunk 1 above)
Comment (r4d6) [reply code/game/objects/items/weapons/tools/hammer.dm:115]: You're the one who removed the r-wall part.
Trilby's asking to remove a superfluous space.

Comment (cdb-is-not-good) [reply code/game/objects/items/weapons/tools/hammer.dm:115]: I'm dumb as fuck and gay too

**Pull request merged**